workflows/resilient_service/resources/server.py

```diff
@@ -91,7 +91,7 @@ def do_GET(self):
         elif self.path == '/healthcheck':
             response_code = 200
             success = True
-            if parameters[health_check] == 'deep':
+            if parameters.get(health_check, '') == 'deep':
```

Comment:
Okay to recover from this, but I would log this situation, because the parameters "should" all be set up with default values before this is called and it would be good to know something's off.

Reply:
Updated